net/http/client.cpp

```diff
@@ -47,6 +47,13 @@ class PooledDialer {
             resolver(new_default_resolver(kDNSCacheLife)) {
     }
 
+    PooledDialer(TLSContext *_tls_ctx) :
```

Comment:
No need to add a new constructor; simply add a default parameter to the existing constructor. The constructor is involved in the initialization of multiple members, and it is not advisable to maintain multiple versions.